File: app/src/main/java/com/example/village/screen/signup/SignupActivity.java
package com.example.village.screen.signup;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.example.village.R;

public class SignupActivity extends AppCompatActivity {

    SignupFragment1 signupFragment1;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_signup);
        signupFragment1 = new SignupFragment1();
        getSupportFragmentManager().beginTransaction()
                .replace(R.id.fragment, signupFragment1)
                .commit();
    }


}

File: app/src/main/java/com/example/village/screen/signup/SignupActivity.kt
package com.example.village.screen.signup

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.fragment.app.Fragment
import com.example.village.R

class SignupActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        var transition = supportFragmentManager.beginTransaction()
                .replace(R.id.fragment, SignupFragment1())
        transition.commit()

    }

}